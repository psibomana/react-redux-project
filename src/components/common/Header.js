import React from 'react';
import { Link } from 'react-router-dom';

const Header = () => {
  return (
    <nav className="navbar navbar-expand-lg navbar-light bg-light">
      <div className="navbar-header">
        <Link className="navbar-brand" to="/">Pluralsight Administration</Link>
      </div>
      <div className="navbar navbar-expand-lg navbar-light bg-ligh">
        <Link to="/" className="nav-item nav-link active">Home</Link>
        <Link to="/courses" className="nav-item nav-link">Courses</Link>
        <Link to="/authors" className="nav-item nav-link">Authors</Link>
        <Link to="/about" className="nav-item nav-link">About</Link>
      </div>
    </nav>
  );
};

export default Header;
