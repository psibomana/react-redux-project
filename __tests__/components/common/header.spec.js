import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from '../../../src/components/common/header';

describe('The Header Component', () => {
  it('should render as expected', () => {
    const header = shallow(<Header />);
    expect(toJson(header)).toMatchSnapshot();
  })
});
