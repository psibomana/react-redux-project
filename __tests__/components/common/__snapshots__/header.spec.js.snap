// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Header Component should render as expected 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-light bg-light"
>
  <div
    className="navbar-header"
  >
    <Link
      className="navbar-brand"
      replace={false}
      to="/"
    >
      Pluralsight Administration
    </Link>
  </div>
  <div
    className="navbar navbar-expand-lg navbar-light bg-ligh"
  >
    <Link
      className="nav-item nav-link active"
      replace={false}
      to="/"
    >
      Home
    </Link>
    <Link
      className="nav-item nav-link"
      replace={false}
      to="/courses"
    >
      Courses
    </Link>
    <Link
      className="nav-item nav-link"
      replace={false}
      to="/authors"
    >
      Authors
    </Link>
    <Link
      className="nav-item nav-link"
      replace={false}
      to="/about"
    >
      About
    </Link>
  </div>
</nav>
`;
